### DISCUSSION TITLE
location pane 'cursor' color?

### DISCUSSION THREAD

**Discussion by arrowd43** [WinMerge/winmerge#1704] [General]
If you look at the attached screenshot, you'll see that the 'cursor' on the location pane is barely visible. I don't know if this item has a generally known name - it's pointed out with a purple arrow. I tried to find a way to change the color to something that would stand out, but I am unable to find it. Is there such a color setting? If not, perhaps that could be added??
![winmerge cursor](https://user-images.githubusercontent.com/40767101/216690590-639a1c63-f951-4294-b300-d24a74916046.jpg)

**Comment by sdottaka** [comment]
The area is displayed with the color of the image in that area darkened.
For this reason, the color is not defined and cannot be changed at this time.
In the future, it may be possible to adjust the darkness.

**Comment by arrowd43** [comment]
Thanks for the explanation. Darker would be better.
